src/devices/MAX31856/MAX31856.cs

```diff
@@ -48,18 +48,17 @@ public bool TryGetTemperature(out Temperature temperature)
                 temperature = ReadTemperature(out byte[] spiOutputData);
                 return true;
             }
-            catch (IOException e)
+            catch (IOException)
             {
                 // The tempearture is set to 1 million if there is an exception
                 temperature = Temperature.FromDegreesCelsius(1_000_000);
                 // Write error message to council
-                Console.WriteLine("IOException :" + e.Message);
                 return false;
             }
         }
 
         /// <summary>
-        ///  Reads the temperature from the sensor
+        ///  Reads the temperature from the Cold-Junction sensor
         /// </summary>
         /// <returns>
         ///  Temperature in degrees celsius
```

Comment:
I would got for the long name here as well GetColdJunctionTemperature()

Reply:
updated thank oyu